index.bs

```diff
@@ -1224,25 +1614,85 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
+        1. Set |operand|.{{MLOperand/[[name]]}} to |name|.
+        1. Make a request to the underlying platform to:
+            1. Create an [=implementation-defined=] platform input operand |operandImpl| given |descriptor|.
+            1. Store a reference of |operandImpl| in |operand|.{{MLOperand/[[operand]]}}.
+            1. Register |operand| as an input.
+    1. Return |operand|.
+  </div>
+</details>
+
+### The build() method ### {#api-mlgraphbuilder-build}
+Build a composed graph up to a given output operand into a computational graph, asynchronously or synchronously.
+
+#### The {{MLGraphBuilder/build(outputs)}} method #### {#api-mlgraphbuilder-build-outputs}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/build(outputs)}} steps are:
+  </summary>
+  <div algorithm=build-outputs class=algorithm-steps>
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. Let |promise| be [=a new promise=].
+    1. Return |promise| and run the following steps [=in parallel=].
+    1. Return the result of invoking {{MLGraphBuilder/buildSync(outputs)}} given |outputs|.
+        1. If that throws, re-throw the error and stop.
+  </div>
+</details>
+
+#### The {{MLGraphBuilder/buildSync(outputs)}} method #### {#api-mlgraphbuilder-buildsync-outputs}
+<details open>
+  <summary>
+    The {{MLGraphBuilder/buildSync(outputs)}} steps are:
+  </summary>
+  <div algorithm=buildsync-outputs class=algorithm-steps>
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. If |outputs| is not an instance of {{MLNamedOperands}}, then throw an "{{TypeError}}" {{DOMException}} and stop.
+    1. For each |element| in |outputs|:
+        1. If |element|.key is not a [=string=], then throw an "{{TypeError}}" {{DOMException}} and stop.
+        1. If |element|.value is not an instance of {{MLOperand}}, then throw an "{{TypeError}}" {{DOMException}} and stop.
+    1. If any of the following sub-steps fail, throw an "{{OperationError}}" {{DOMException}} and stop.
+        1. Let |graph| be a new {{MLGraph}}:
+            1. Set |graph|.{{MLGraph/[[context]]}} to [=this=].{{MLGraphBuilder/[[context]]}}.
+            1. Set |graph|.{{MLGraph/[[outputDescriptors]]}} to |outputs|.
+        1. Make a request to the underlying platform to:
+            1. Connect |graph| to a new [=implementation-defined=] graph implementation |graphImpl| given |graph|.
+            1. Store a reference to |graphImpl| in |graph|.{{MLGraph/[[implementation]]}}.
+        1. Make a request to the underlying platform to initialize the graph:
+            1. For each |operand| in |outputs|:
+                1. If |operand| was created as an input by the underlying platform:
+                    1. Add |operand| to |graph|.{{MLGraph/[[inputDescriptors]]}}.
```

Comment:
ditto, the type of |operand| is not compatible to |graph|.{{MLGraph/[[inputDescriptors]]}}.